contrib/net/http/roundtripper.go

```diff
@@ -25,8 +25,11 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		tracer.Tag(ext.HTTPMethod, req.Method),
 		tracer.Tag(ext.HTTPURL, req.URL.Path),
 	}
-	if rate := rt.cfg.analyticsRate; rate > 0 {
-		opts = append(opts, tracer.Tag(ext.EventSampleRate, rate))
+	if rt.cfg.analyticsRate > 0 {
+		opts = append(opts, tracer.Tag(ext.EventSampleRate, rt.cfg.analyticsRate))
+	}
+	if rt.cfg.serviceName != "" {
```

Comment:
Let's use the same logic here, for consistency:
```go
if service := rt.cfg.serviceName; service != "" {
    // ...
```

Reply:
Consistency and readability were the reasons I changed also the first one. If you really prefer that version I can change. I find your version less readable, but maybe it's just me.

Reply:
Yes, please do. Probably `name` is a better name than `service`.